cpp/examples/config.h

```diff
@@ -16,8 +16,42 @@ limitations under the License.
 #include <filesystem>
 #include <string>
 
-#ifndef TEST_CONFIG_H_
-#define TEST_CONFIG_H_
+#ifndef CPP_EXAMPLES_CONFIG_H_
+#define CPP_EXAMPLES_CONFIG_H_
+
+// Define a new macro that is just like the standard C assert macro,
+// except that it works even in optimized builds (where NDEBUG is
+// defined) and it prints the failed assertion to stderr.
+#ifndef ASSERT
+#define ASSERT(x)                                    \
```

Comment:
Could you help to unify the similar `ASSERT` and `ABORT`  in unit tests as well?

Reply:
Thank you for the reminder. Currently, `assert` is only used in the examples.